src/multiselect_contextmenu.js

```diff
@@ -24,11 +27,13 @@ const registerCopy = function(useCopyPasteCrossTab) {
     displayText: function(scope) {
       let workableBlocksLength = 0;
       const workspace = scope.block.workspace;
-      const blockSelection = blockSelectionWeakMap.get(workspace);
-      blockSelection.forEach(function(id) {
+      const dragSelection = dragSelectionWeakMap.get(workspace);
+      dragSelection.forEach(function(id) {
         const block = workspace.getBlockById(id);
-        if (copyOptions.check(block)) {
-          workableBlocksLength++;
+        if (block) {
+          if (copyOptions.check(block)) {
```

Comment:
Merge the conditions together into one branch:

```suggestion
        if (block && copyOptions.check(block)) {
```